### PR TITLE
Add Flatiron Apprenticeship

### PR DESCRIPTION
**Pull request by changangus**
Insert description of PR here

---

## ✅️ By submitting this PR, I have verified the following

- [x] Checked to see if a similar PR has already been opened 🤔️
- [x] Reviewed the contributing guidelines 🔍️
- [x] Added my name to the bottom of the list under the **Credits** section in the `README.md` with a link to my website or GitHub profile 👥️

### CODE REVIEW

File: content/apprenticeships/Flatiron.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+company: "Flatiron Health"
+description: "In this role, you'll work with a software engineering team to help oncologists spend more time with patients or empower scientists to conduct research faster."
+image: "/images/apprenticeships/flatiron.jpeg"
+link: "https://flatiron.com/"
```
Comment (benjaminmodayil) [inline content/apprenticeships/Flatiron.md:5]: Can you change this to a direct link to the listing or program? This is a link I found:
https://flatiron.com/careers/open-positions/2451150

File: content/apprenticeships/Flatiron.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+---
+company: "Flatiron Health"
+description: "In this role, you'll work with a software engineering team to help oncologists spend more time with patients or empower scientists to conduct research faster."
+image: "/images/apprenticeships/flatiron.jpeg"
```
Comment (benjaminmodayil) [inline content/apprenticeships/Flatiron.md:4]: Can you update this `image` line to:

```
image: "flatiron.jpg" 
```

It seems I need to update the documentation as adding a path to the image isn't valid, but just the image name and extension is valid.

Also you had an "e" in your file extension when you have the file saved as `.jpg`

**Review by benjaminmodayil — CHANGES_REQUESTED** [review]
Hi @changangus  I left two comments for you. Let me know if I can help in any way. I would make these small changes for you if I could, but I don't have permissions to submit changes to other forks of the repo.

**Comment by changangus** [comment]
Hey Ben,

Just made those changes, let me know if there is anything else that I might have missed!

**Review by benjaminmodayil — APPROVED** [review]

**Comment by benjaminmodayil** [comment]
Looks good. Sorry for the late reply @changangus

**Pull request merged**